src/components/modules/puzzles/puzzlesBanner/puzzlesBannerStyles.ts

```diff
@@ -0,0 +1,144 @@
+import { css } from '@emotion/core';
+import { between, from, until } from '@guardian/src-foundations/mq';
+import { headline } from '@guardian/src-foundations/typography/cjs';
+import { neutral, lifestyle } from '@guardian/src-foundations/palette';
+import { space } from '@guardian/src-foundations';
+
+export const banner = css`
+    html {
+        box-sizing: border-box;
+    }
+    *,
+    *:before,
+    *:after {
+        box-sizing: inherit;
+    }
+    box-sizing: border-box;
+    width: 100%;
+    background-color: ${lifestyle[300]};
+    color: ${neutral[100]};
+    border: 2px solid ${neutral[0]};
+
+    ${until.tablet} {
+        display: flex;
+        flex-direction: column;
+        align-items: flex-end;
+    }
+`;
+
+export const bannerContents = css`
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+
+    ${from.tablet} {
+        flex-direction: row;
+        height: 260px;
+    }
+
+    ${from.desktop} {
+        height: 344px;
+    }
+`;
+
+export const squaresContainer = css`
+    position: relative;
+    display: flex;
+    justify-content: flex-end;
+    height: 100%;
+    width: 100%;
+    max-width: 690px;
+
+    ${until.tablet} {
+        position: absolute;
+        bottom: 50px;
+        right: 0;
+    }
+
+    ${from.desktop} {
+        padding-right: 44px;
+    }
+`;
+
+export const headingSection = css`
+    max-width: 500px;
+    margin-right: ${space[6]}px;
+`;
+
+export const heading = css`
+    ${headline.large({ fontWeight: 'bold' })};
+    margin: 0 0 ${space[6]}px;
+
+    ${between.tablet.and.desktop} {
+        ${headline.small({ fontWeight: 'bold' })}
+    }
+
+    ${from.leftCol} {
+        font-size: 66px;
+    }
+`;
+
+export const appStoreButtonContainer = css`
+    display: flex;
+    flex-direction: column;
+
+    a {
+        margin-bottom: ${space[2]}px;
+    }
+
+    ${from.leftCol} {
+        flex-direction: row;
+
+        a:not(:last-of-type) {
+            margin-right: ${space[4]}px;
+        }
+    }
+`;
+
+export const collapseButtonContainer = css`
+    & > * {
```

Comment:
What is the ampersand doing in this context? Are you selecting all children of the `collapseButtonContainer`? Totally aware that this may just be my ignorance, and I'd love to learn!

Reply:
The ampersand means essentially the same thing as in Sass- it refers to the current selector. > is the [child combinator selector](https://developer.mozilla.org/en-US/docs/Web/CSS/Child_combinator), which means 'select the _immediate_ children of the previous selector which match the following selector'- so this is selecting all immediate children of the element which `collapseButtonContainer` gets applied to.